src/test/java/bio/terra/grammar/GrammarTest.java

```diff
@@ -159,8 +167,14 @@ public void testSynapseTranslate_NoTableNameAlias() {
         Query.parse(
             "SELECT datarepo_row_id FROM datasetName.vocabulary WHERE vocabulary_id IN ('1')");
     String expectedQuery =
-        "SELECT datarepo_row_id FROM (SELECT * FROM\nOPENROWSET(\n  BULK 'metadata/parquet/vocabulary/*/*.parquet',\n  DATA_SOURCE = 'sourceDatasetDataSourceName1',\n  FORMAT = 'parquet') AS inner_alias927641339) AS alias927641339\n WHERE vocabulary_id IN ( '1' )";
-    String translatedQuery = query.translateSql(synapseVisitor);
+        QueryTestUtils.collapseWhiteSpace(
+            """
+      SELECT datarepo_row_id FROM (SELECT * FROM OPENROWSET(
+        BULK 'metadata/parquet/vocabulary/*/*.parquet',
+        DATA_SOURCE = 'sourceDatasetDataSourceName1',
+        FORMAT = 'parquet') AS inner_alias927641339) AS alias927641339
+      WHERE vocabulary_id IN ( '1' )""");
+    String translatedQuery = QueryTestUtils.collapseWhiteSpace(query.translateSql(synapseVisitor));
     assertThat("Translation is correct", translatedQuery, equalTo(expectedQuery));
```

Comment:
I didn't see it before, but there's a hamcrest matcher that ignores whitespace, which could work here instead of using a util method to remove it.
```suggestion
    assertThat("Translation is correct", translatedQuery, equalToIgnoringWhiteSpace(expectedQuery));
```